zerver/tests/test_events.py

```diff
@@ -2778,13 +2779,35 @@ def test_notify_realm_export(self) -> None:
         with mock.patch('zerver.lib.export.do_export_realm',
                         return_value=create_dummy_file('test-export.tar.gz')):
             events = self.do_test(
-                lambda: print(self.client_post('/json/export/realm').content),
```

Comment:
I'm guessing this was just for debugging or something? Looks like I was having trouble by the simple fact of forgetting to assign admin status...

Reply:
Yes, we don't need the print there.